editor/block-icon/index.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
>Interesting; why did you choose to put it in "editor"?

It's the editor using it. My thinking is: components in `blocks` are used in block implementations, components in `editor` are specific to editor rendering (like here), and components in `components` are shared between contexts.

Where did you have in mind?